src/components/Instructions.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import { CIRCLE_VALUE, NEUTRAL_VALUE } from '../constants/game';
+
+const CIRCLE_SRC = '/static/circle.png';
+const CROSS_SRC = '/static/cross.png';
+
+const Instructions = ({ team, isPlaying, winner }) => {
```

Comment:
Don't forget the prop-types for your components

Reply:
Done